nodes/srl/srl.go

```diff
@@ -161,8 +165,13 @@ func (s *srl) Deploy(ctx context.Context) error {
 }
 
 func (s *srl) PostDeploy(ctx context.Context, ns map[string]nodes.Node) error {
+	if s.cfg.StartupConfig == "" {
```

Comment:
the postdeploy action where we add config on top of the factory one is only needed if no startup config is present.

also we need to make sure that if the config inside the nodeDir is present we also don't call this postdeploy action, as the config is nodeDir should already have all the needed stuff.
**TODO: THIS IS NOT TAKEN INTO ACCOUNT YET**

Reply:
resolved in eb11522